### DISCUSSION TITLE
UI bug in iOS app

### DISCUSSION THREAD

**Discussion by capiet** [community/community#15653] [Mobile] [answered]
Hello. There is the interface bug in the Notifications section. The filter elements are attached to the top and space between them and the bottom border is bigger than at the top.
![822D9FF2-9185-44F2-B451-6C875002F804](https://user-images.githubusercontent.com/79054211/165389997-51873443-0eff-42ce-a9a3-ea6b00435ca5.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi, thanks for the feedback! Can you consistently reproduce this? Also what version is this?

**Reply by capiet** [reply]
@stevepopovich Hello, I can consistently reproduce this bug on iPhone XR, iOS 14.8.1. Application version is 1.62.1 (66498926).

**Reply by stevepopovich** [reply]
Hi! As we will be deprecating this iOS version this year (iOS 14) and it only happens on this device type, we are marking this as a `wont fix`. Thanks!

**Reply by 3m4r5** [reply]
@stevepopovich it also happens to me on iPhone 8 Plus on iOS 14.7.1
